### DISCUSSION TITLE
Does Coder have Redis and SQS support?

### DISCUSSION THREAD

**Discussion by nathanielDuro** [coder/coder#12148] [General] [answered]
Is it possible to configure Coder to use external Redis as the cache layer? Does Coder support message queuing service like SQS?
Our current setup: Coder is installed in Ubuntu EC2 and for our workspace deployment we use Docker.

**Comment by matifali** [comment]
>  Coder to use external Redis as the cache layer? Does Coder support message queuing service like SQS?

Where do you want to use these? I assume you want your workspaces to connect to these.
Coder does not need Radis and SQS for its own functionality.

**Comment by ammario** [comment, marked as answer]
What exactly do you want to cache? We do a great deal of caching type work within Postgres at the moment, notably when computing insights and statistics.

Since we're self-installed we greatly value deployment simplicity. It would take a very good reason for us to add another service to the mix.
